Reapply "LPD-63798 Inline"

This reverts commit 71325394710ef6cda2dd59fba39e5f3136b6d348.

sandrodw3 opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/84694. [See changes here.](https://github.com/sandrodw3/liferay-portal/compare/a72178215066~4...a72178215066) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

RoselaineMarquesMontero committed:
LPD-62162 Make LayoutPageTemplateCollection indexable

RoselaineMarquesMontero committed:
LPD-62162 Add IndexerReindexTest

victorg1991 committed:
LPD-62003 Allow setting inputReadOnly in the configuration of the inputs

victorg1991 committed:
LPD-62003 Select if the inputs should be read only

# breaking

## What modules/apps/layout/layout-api/src/main/java/com/liferay/layout/manager/FormManager.java

Allow rendering the inputs as read only when filling the form

## Why

For side by side translation view we need one of the sides to be represented as a read only form

----

victorg1991 committed:
LPD-62003 Adapt code

victorg1991 committed:
LPD-62003 Rename

victorg1991 committed:
LPD-62003 Add master layout for translation

victorg1991 committed:
LPD-62003 Add Struts action for translating that will create display page if not created

victorg1991 committed:
LPD-62003 Add option to translate content

victorg1991 committed:
LPD-62003 Extract constant

victorg1991 committed:
LPD-62003 Add reset translation display page struts action

victorg1991 committed:
LPD-62003 Reset translation page after publish

victorg1991 committed:
LPD-62003 Add Test

victorg1991 committed:
LPD-62003 Update test

victorg1991 committed:
LPD-62003 Fix call

victorg1991 committed:
LPD-62003 Baseline

victorg1991 committed:
LPD-62003 Add CSS class to the main form to differentiate it

victorg1991 committed:
LPD-62003 Don't create constant file

achaparro committed:
LPD-62162 Not needed

achaparro committed:
LPD-62162 Use existing group due to performance reasons

achaparro committed:
LPD-62162 Test the update case too

achaparro committed:
LPD-62162 SF

brianchandotcom committed:
LPD-62003 SF

achaparro committed:
LPD-62162 Remove Fixture

achaparro committed:
LPD-62162 Test also deleting using the service. It improves performance since DataGuard does nothing

achaparro committed:
LPD-62162 Inline

brianchandotcom committed:
LPD-62162 Rename

brianchandotcom committed:
LPD-62162 Rename

brianchandotcom committed:
LPD-62162 Rename

brianchandotcom committed:
LPD-62162 auto SF

ling-alan-huang committed:
LPD-63871 Auto SF

liferay-translation committed:
LPS-77699 Update Translations

victorg1991 committed:
LPD-64840 Allow disabling portlet rendering

victorg1991 committed:
LPD-64840 We don't need to render portlet to obtain the editable fields

victorg1991 committed:
LPD-64840 baseline

victorg1991 committed:
LPD-64840 Add suppression

sandrodw3 committed:
LPD-62891 Update list of tests that run in relevant when applying changes in fragment

sandrodw3 committed:
LPD-64840 Add test with CMS basic functionality to smoke

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
ci:test:relevant

liferay-continuous-integration commented:
Test suite relevant has been triggered on http://test-1-39

liferay-continuous-integration commented:
<html><h3>:x: ci:test:relevant - 0 out of 1 jobs passed in 3 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/2e1dd3cea588015764bf5a5005a399bd6dc6204a">2e1dd3cea588015764bf5a5005a399bd6dc6204a</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/f7563c9bdefe1217124a74859cc5c3d7eaf1a93c">f7563c9bdefe1217124a74859cc5c3d7eaf1a93c</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/#/project/35392/routines/590307/build/303272499">EE Development Acceptance (master) - 1616 - 2025-08-27[08:34:03]</a></p><details><summary><strong>ci:test:relevant - 0 out of 1 jobs PASSED</strong></summary><div><h4>1 Failed Jobs:</h4><ul><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/14544/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li></ul></div></details><h5>For more details click <a href="https://test-1-39.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14544/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/14544/">test-portal-acceptance-pullrequest(master)</a></h5><div><h6>Job Results:</h6><p>0 Jobs Passed.<br/>1 Job Failed.</p></div><pre>A CI failure has occurred. CI Infrastructure has been notified. Please try again later or contact CI Infrastructure for an update.</pre></div></li></ol><h4>For upstream results, click <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">here</a>.</h4></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/14544">test-portal-acceptance-pullrequest(master)#14544</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-39.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/14544/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/sandrodw3/liferay-portal/pull/199">sandrodw3#199</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest(master)/14544/">test-portal-acceptance-pullrequest(master)#14544</a><br/></div></div>

sandrodw3 commented:
ci:test:relevant

liferay-continuous-integration commented:
Test suite relevant has been triggered on http://test-1-26

liferay-continuous-integration commented:
<h1>Build completed.</h1><p>Jenkins URL: <a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/15727/">test-portal-acceptance-pullrequest(master)</a></p><p>The <code>relevant</code> test suite did not execute tests because all modified files were excluded by the <b>modified.files.global.excludes</b> property in <code>test.properties</code>.</p><p>Please use <code>ci:test:sf</code> and/or <code>ci:forward</code> for this pull request to be forwarded or eligible for forwarding.</p>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/15727">test-portal-acceptance-pullrequest(master)#15727</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-26.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/15727/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/sandrodw3/liferay-portal/pull/199">sandrodw3#199</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/#/project/35392/routines/45357/build/311569052">[master] ci:test:relevant - sandrodw3 &gt; sandrodw3 - PR#199 - 2025-09-08[08:43:44]</a><br/></div><div><strong>Testray Build ID:</strong><span></span><code>311569052</code><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest(master)/15727/">test-portal-acceptance-pullrequest(master)#15727</a><br/></div></div>

victorg1991 committed:
LPD-64840 baseline
